docs/DeveloperGuide.md

```diff
@@ -271,6 +271,112 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `App-Ointment` and the **Actor** is the `User`, unless specified otherwise)
 
+### UC01: Add an appointment
+**MSS**
+1. User adds an appointment.
+2. App-Ointment adds a new appointment to the appointment schedule.
+
+**Extensions**
+* **1a.** App-Ointment detects an invalid command from the user.
+    * **1a1.** App-Ointment prompts user that the command is unrecognised and automatically registers the `help` command.<br>
+    * **1a2.** User enters valid command `add ...`<br>
+    Use case resumes from step 1.
+    
+* **1b.** App-Ointment detetcs an invalid subcommand format.
+    * **1b1.** App-Ointment prompts user that syntax is incorrect and displays the expected format.<br>
+    * **1b2.** User enters correct subcommand syntax.<br>
```

Comment:
might want to consider changing to something like this?
```1a2 user enters a new command
steps 1a1 to 1a2 are repeated until command entered is correct/free from errors.
```
same thing for the other loop-style/invalid input by user use cases